src/map/battle.cpp

```diff
@@ -6170,6 +6174,14 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 			case AB_RENOVATIO:
 				ad.damage = status_get_lv(src) * 10 + sstatus->int_;
 				break;
+			case NPC_EARTHQUAKE:
+				if (src->type == BL_PC)
+					ad.damage = sstatus->str * 2 + sstatus->rhw.atk;
+				else
+					ad.damage = sstatus->str + status_get_lv(src) + (80 + rnd() % 41) * sstatus->rhw.atk;
```

Comment:
Not entirely sure about the player damage but I think this should be something like this:

```
				if (src->type == BL_PC)
					ad.damage = sstatus->str * 2 + battle_calc_weapon_attack(src, src, skill_id, skill_lv, mflag).damage;
				else
					ad.damage = battle_calc_base_damage(src, sstatus, &sstatus->rhw, sc, tstatus->size, 0);
```